.gitignore

```diff
@@ -0,0 +1,7 @@
+*.gcompcache
+*.lvprojectcache
+*.niaddoncache
+*.lvcodedb
+*.aliases
+*.lvlps
+*.gcompcache-journal
```

Comment:
Ah I see. Did you enable Github Pages for the master branch then?

If you do that would be good to add an index.html at the repo root that link to the different examples.

I don't think we have any files starting with an underscore but just to be safe you may also want to add an empty .nojekyll file to the repo root so Github Pages doesn't try to get all magic-y on you (if you enable Github Pages on master).